actiontext/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Fix an issue that caused the content layout to render multiple times when a
+    rich_text field was updated.
+
+    *Jacob Herrington*
+
+*   Fix an issue with how nested lists were displayed when converting to plain text
+
+    *Matt Swanson*
+
+*   Allow passing in a custom `direct_upload_url` or `blob_url_template` to `rich_text_area_tag`.
+
+    *Lucas Mansur*
```

Comment:
Not sure what's going on here, I rebased with main and pulled these in. Any idea where these came from @jonathanhefner?

Reply:
These seem to be coming from a [previous CHANGELOG](https://github.com/rails/rails/blob/7-0-stable/actiontext/CHANGELOG.md#rails-700rc1-december-06-2021).  I'm not sure how they got pulled in, but it should be safe to remove them.